ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -267,16 +278,27 @@ public void requestBannerAd(Context context,
     }
 
     private VungleListener mVungleBannerListener = new VungleListener() {
+
         @Override
-        void onAdEnd(String placement, boolean wasSuccessfulView, boolean wasCallToActionClicked) {
+        void onAdClick(String placementId) {
             if (mMediationBannerListener != null) {
-                if (wasCallToActionClicked) {
-                    // Only the call to action button is clickable for Vungle ads. So the
-                    // wasCallToActionClicked can be used for tracking clicks.
-                    mMediationBannerListener.onAdClicked(VungleInterstitialAdapter.this);
-                    mMediationBannerListener.onAdOpened(VungleInterstitialAdapter.this);
-                    mMediationBannerListener.onAdClosed(VungleInterstitialAdapter.this);
-                }
+                // Only the call to action button is clickable for Vungle ads.
+                mMediationBannerListener.onAdClicked(VungleInterstitialAdapter.this);

```

Comment:
logic differs from https://github.com/Vungle/googleads-mobile-android-mediation/pull/35/files#diff-e3eb809fec019433eb4442f5941b9e9fR288

Reply:
yes, when 6.6.x propagate PR merged to develop will update this PR then.